addons/fluxcd/definitions/helm-release-def.yaml

```diff
@@ -109,6 +109,8 @@ spec:
         	repoType: *"helm" | "git" | "oss"
         	// +usage=The interval at which to check for repository/bucket and relese updates, default to 5m
         	pullInterval: *"5m" | string
+            // +usage=The  Interval at which to reconcile the Helm release, default to 30s
```

Comment:
Pay attention to formatting alignment.

Are 30 seconds too long?

Reply:
Have thought about this. In fact fluxcd recommend 1m is good https://fluxcd.io/docs/guides/helmreleases/ ,I concern about that too short interval will have other problems.